src/modules/roc_core/target_bsd/roc_core/format_tid.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#if defined(__FreeBSD__)
+#include <pthread_np.h>
+
+#include "roc_core/format_tid.h"
+
+namespace roc {
+namespace core {
+
+int format_tid() {
+    return pthread_getthreadid_np();
+}
```

Comment:
I was mistaken. MacOS and BSD have different functions: it's `pthread_threadid_np` on MacOS (defined in `pthread.h`) and `pthread_getthreadid_np` on BSD (defined in `pthread_np.h`). So we can't use the same target directory for MacOS and BSD...

We don't have BSD support yet. We will add it in #284. Thus, let's remove target_bsd for now. Let's add only target_linux. We already have target_darwin for MacOS.

format_tid from target_linux will use syscall() and format_tid from target_darwin will use pthread_threadid_np.